sn_networking/src/record_store.rs

```diff
@@ -472,12 +472,7 @@ impl NodeRecordStore {
         let record_key = PrettyPrintRecordKey::from(&r.key).into_owned();
         trace!("PUT a verified Record: {record_key:?}");
 
-        // notify fetcher
-        if let Err(error) = self.prune_records_if_needed(&r.key) {
-            let cmd = SwarmCmd::RemoveFailedLocalRecord { key: r.key };
-            send_swarm_cmd(self.swarm_cmd_sender.clone(), cmd);
-            return Err(error);
-        }
+        self.prune_records_if_needed(&r.key)?;
```

Comment:
'r' should not be a variable name. It's cryptic. We need to label variables correctly. 
I will approve but please add a descriptive name